### DISCUSSION TITLE
Cardinal VST3 crash in Studio One

### DISCUSSION THREAD

**Discussion by ArkoS2000** [DISTRHO/Cardinal#684] [Q&A]
I am trying to install Cardinal VAT in studio one but as I try to run the VST3 it crashes my DAW. 
Can anyone help?

**Comment by dromer** [comment]
Which version of Cardinal and on which OS?

**Reply by ArkoS2000** [reply]
Windows 10, Cardinal v24.05

**Reply by ArkoS2000** [reply]
Okay, I tried running in Bitwig and this happend. Not sure what's the problem

![Screenshot 2024-07-03 103639](https://github.com/DISTRHO/Cardinal/assets/132554191/7b86bd7c-1d49-4b4e-8620-c45d1e514941)

**Comment by falkTX** [comment]
I once tried to reproduce this with the demo version but studio one but couldn't.

Is it related to the version of Studio One perhaps? which one are you using?

**Reply by ArkoS2000** [reply]
Studio One 6.0.0

**Reply by ChapelHeel66** [reply]
I have the same issue with Windows 10 and Studio One 5.4.   The VST2 of Cardinal Synth and Cardinal FX run fine, but the full Cardinal (which is VST3 only), crashes Studio One.   Sometimes it crashes on instantiation and I get and error message.  In that case, I have to manually shut down S1 through Task Manager.  Other times it opens to the default modules and I can add modules.  But if I try to load a saved patch or just start a new one, S1 crashes and S1 automatically restarts.

I need the full version for the multiple audio outs.  So, here is what I'm doing as a workaround.

I sometimes use a plugin wrapper made by Kushview (called Elements) to test ideas.  Normally I do this in Elements standalone because it starts up far more quickly than Studio One.  However, Element can also load into your DAW as a VST2 or VST3 plugin (instrument).

So, I load Element VST3 as an instrument in Studio One, and then from _within_ Element, I load the full Cardinal VST3.  (I also understand that some ppl use this as a trick in DAWs that do not support VST3...they load Element as a VST2 instrument in their DAW, and then load the desired VST3 inside Element).

Anyway, Element should show Cardinal as having 16 audio outs (8 L R).  Element has its own audio out, so within Elements you just need to connect the corresponding out from the Cardinal icon to the Element audio out icon.  However, initially the Element icon for audio out will only have two nodes.  So, in Studio One do what you normally do for multi-outs on software instruments:  In Console view choose Instruments, right click on Element and choose "Expand" to see the possible outs.  Then select the ones you want to use.  When you go back to Element, you will now see additional nodes that you can connect to the corresponding out nodes from Cardinal.  

This is clunky, but it will do until someone WAY smarter than me figures out why Studio One and full Cardinal VST3 do not like each other.
